modules/pool/invest/components/PoolInvestCustom.tsx

```diff
@@ -65,7 +65,7 @@ export function PoolInvestCustom({ onShowPreview }: Props) {
                 isFullWidth
                 mt="8"
                 onClick={onShowPreview}
-                isDisabled={(totalTokenAmounts <= 0 || hasHighPriceImpact) && !acknowledgeHighPriceImpact}
+                isDisabled={hasAllZeroTokenAmounts || (hasHighPriceImpact && !acknowledgeHighPriceImpact)}
```

Comment:
You're still missing a check that the amount input is more than the user balance